src/braket_container.py

```diff
@@ -149,14 +150,27 @@ def kick_off_customer_script(entry_point : str) -> multiprocessing.Process:
         str_module, _, str_method = entry_point.partition(":")
         customer_module = importlib.import_module(str_module)
         customer_method = getattr(customer_module, str_method)
-        customer_code_process = multiprocessing.Process(target=customer_method)
+
+        process_kwargs = {"target": customer_method}
+
+        hp_file = os.environ["AMZN_BRAKET_HP_FILE"]
+        with open(hp_file) as f:
+            hyperparams = json.load(f)
+
+        try:
+            inspect.signature(customer_method).bind(**hyperparams)
```

Comment:
I thought that hyperparamters were converted to string by SageMaker, so you may have a float parameter, but you actually have to convert it to float, no?

Reply:
unit test?

Can you test the boundary cases (function has no parms, too many params, etc.)?